evadb/parser/create_statement.py

```diff
@@ -183,3 +183,42 @@ def __hash__(self) -> int:
                 self.query,
             )
         )
+
+
+class CreateDatabaseStatement(AbstractStatement):
```

Comment:
Should the CreateDatabaseStatement inherit SKIP_BINDER_AND_OPTIMIZER_STATEMENTS?

Reply:
Figured it out how it works.